styles.css

```diff
@@ -705,4 +705,96 @@ textarea::placeholder {
     letter-spacing: 0.03em;
     color: #396df2;
   }
+
+  .card-reverse {
+    flex-direction: row-reverse;
+  }
+
+  .about-me {
+    height: 700px;
+    padding-top: 100px;
+  }
+
+  .about-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .about-container div {
+    width: 50%;
+  }
+
+  .about-title {
+    width: 100px;
+  }
+
+  .connect {
+    margin: 40px 0;
+  }
```

Comment:
As stated by the previous reviewer, the `let's connect`  is still messed up with the about myself paragraph as shown below :point_down: Kindly fix this to produce a good and responsive user experience. :+1: 


![Screenshot from 2023-01-21 22-56-56](https://user-images.githubusercontent.com/65233860/213891482-2599d21f-c312-4906-bf82-fa592ab3753f.png)